apps/web/next.config.js

```diff
@@ -1,7 +1,7 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
     output: 'standalone',
-    basePath: process.env.NODE_ENV === "production" ? "/merch" : "",
+    basePath: "/merch",
```

Comment:
This should be left as is because the app can still be run outside the containers